projects/plugins/boost/app/lib/minify/functions-helpers.php

```diff
@@ -21,6 +22,9 @@ function jetpack_boost_minify_cache_buster() {
  * @param int $file_age The age of files to purge, in seconds.
  */
 function jetpack_boost_legacy_minify_cache_cleanup( $file_age = DAY_IN_SECONDS ) {
+	// Explicitly cast to int as do_action() can pass a non-int value. https://core.trac.wordpress.org/ticket/14881
+	$file_age = is_int( $file_age ) ? $file_age : DAY_IN_SECONDS;
```

Comment:
I am a bit confused about this. Don't we want to do `(int) $file_age` instead?
```suggestion
	$file_age = (int) $file_age;
```

Reply:
My comment was misleading actually, so I've updated it.
`$file_age` becomes an empty string when doing an action that calls `jetpack_boost_legacy_minify_cache_cleanup`, which is the case with the network cron. So essentially:
```
add_action( 'jetpack_boost_minify_cron_cache_cleanup', 'jetpack_boost_legacy_minify_cache_cleanup' );
do_action( 'jetpack_boost_minify_cron_cache_cleanup' );
```
Due to https://core.trac.wordpress.org/ticket/14881